pennylane/qchem/convert.py

```diff
@@ -1144,10 +1162,10 @@ def _shci_state(wavefunction, tol=1e-15):
     SHCI.outputfile.
 
     Args:
-        wavefunction tuple(array[int], array[str]): determinants and coefficients in physicist notation
+        wavefunction tuple(array[str], array[float]): determinants and coefficients in chemist notation
```

Comment:
Please modify these after using list for determinants.

Reply:
Done.